contracts/interfaces/IStableDebtToken.sol

```diff
@@ -35,15 +35,15 @@ interface IStableDebtToken is IInitializableDebtToken {
   /**
    * @dev Emitted when new stable debt is burned
    * @param from The address from which the debt will be burned
-   * @param amount The amount being burned (user entered amount - balance increase from interest)
-   * @param currentBalance The current balance of the user
-   * @param balanceIncrease The the increase in balance since the last action of the user
+   * @param value The amount being burned (user entered amount - balance increase from interest)
+   * @param currentBalance The balance of the user based on the previous balance and balance increase from interest
+   * @param balanceIncrease The increase in balance since the last action of 'from'
    * @param avgStableRate The next average stable rate after the burning
    * @param newTotalSupply The next total supply of the stable debt token after the action
    **/
   event Burn(
     address indexed from,
```

Comment:
Not having a `target` parameter in the Burn event of the `StableDebtToken` and having it in the corresponding event of `VariableDebtToken` makes little sense to me.
I would be in favor of adding an `address indexed target` as "The address that will receive the underlying, if any"

Reply:
Discarded for compatibility: adding another parameter would change the event signature